server/package.json

```diff
@@ -9,6 +9,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "chai": "^4.3.4",
+    "chai-http": "^4.3.0",
```

Comment:
Add when you have tests! Right now we have 0 tests, based on what you have in the future add them on a need basis. In this manner, we might end up adding a lot of stuff we probably might never need.

Reply:
Have you compared mocha/chai with jest? Why would you prefer them over jest? https://jestjs.io/ check check?

Reply:
Got it, have read about both of them now and I remember using Jest for endpoint testing. Jest is preferable for small-medium scale projects as we don't have to setup other libraries to work with it while mocha is suitable for more advanced use-cases as it has more features and dependencies. Jest should be fine for us and I'll add it later when I'm writing tests.

Reply:
Henceforth, for any spike (research), when you do search and learn things eg. why jest over chai or chai over jest, also share resources that you're referring to. There are a lot of blogs/resources which can push us either way, it's important to know and understand how and where you drive conclusions from.